src/archivist/services/documentDeclaration.js

```diff
@@ -12,4 +12,15 @@ export default class DocumentDeclaration {
   get isMultiPage() {
     return this.pages.length > 1;
   }
+
+  toJSON() {
```

Comment:
I think `toJSON` is misnamed because it not only converts the representation to JSON but also renames its attributes to be the same as those found in service declarations.
What about toPersistenceFormat?